docs/agent/changes.md

```diff
@@ -192,17 +192,12 @@ gauge(self, name, value, tags=None, hostname=None, device_name=None)
 The APM agent (also known as _trace agent_) is shipped by default with the
 Agent 6 in the Linux, MacOS and Windows packages.
 
-Similar to the Agent 5, the APM agent is enabled by default. To disable it, set
-`apm_enabled` to `false` in the main agent configuration (`datadog.yaml`).
-
-_Optional_: If you need to use apm-specific configuration options (i.e. options that would be
-specified under the `[trace.config]`, `[trace.sampler]` and `[trace.receiver]` in
-the former `datadog.conf` file), specify them in `trace-agent.conf` under the agent's
-configuration directory (`/etc/datadog-agent/` on Linux). This file should be
-INI-formatted, similar to the former `datadog.conf` file. See the `trace-agent.conf.example`
-file for an example configuration file.
+The process agent is not enabled by default. To enable the check you can update your `datadog.yaml` file to add the following:
```

Comment:
this should read `APM agent (also known as _trace agent_)`, and we should mention that it's enabled by default on Linux (but not on Windows)